packages/react/src/diagrams/CardNode/CardNode.tsx

```diff
@@ -67,6 +69,11 @@ CardNode.propTypes = {
 	 */
 	children: PropTypes.node,
 
+	/**
+	 * Provide an optional class to be applied on the outer element
+	 */
+	className: PropTypes.string,
```

Comment:
this could be misleading. Seems like it should be called `additionalClassNames`? and why just 1 additional className? maybe a user would like to add multiple

Reply:
The `className` attribute on a DOM element accepts multiple classes and this naming/use is consistent with carbon-components-react.

For example I could have `<CardNode className="my-custom-class another-class" … />` and it works just the same as a single class.

Reply:
@theiliad what do you think? Is it OK as-is?

I would like to get this wrapped up soon so we can start using it.

Reply:
@AlanGreene I guess so. My only concern now is, what about the rest of the diagram components?

This'll kind of be an inconsistency if we add the classname to just 1 component.

What do you think?

Reply:
I don't have a particular use case in mind for it on any of the other diagram components but would be happy to add it if you think it would be useful.